onnx/defs/data_type_utils.cc

```diff
@@ -6,406 +6,86 @@
-  const char* ptr = data_ + size_ - 1;
-  while (count < size_ && isspace(*ptr)) {
-    ++count;
-    --ptr;
-  }
-
-  if (count > 0) {
-    return RStrip(count);
-  }
-  return false;
-}
-
-bool StringRange::RStrip(size_t size) {
-  if (size_ >= size) {
-    size_ -= size;
-    return true;
-  }
-  return false;
-}
-
-bool StringRange::RStrip(StringRange str) {
-  if (EndsWith(str)) {
-    return RStrip(str.size_);
-  }
-  return false;
-}
-
-bool StringRange::LAndRStrip() {
-  bool l = LStrip();
-  bool r = RStrip();
-  return l || r;
-}
-
-void StringRange::ParensWhitespaceStrip() {
-  LStrip();
-  LStrip("(");
-  LAndRStrip();
-  RStrip(")");
-  RStrip();
-}
-
-size_t StringRange::Find(const char ch) const {
-  size_t idx = 0;
-  while (idx < size_) {
-    if (data_[idx] == ch) {
-      return idx;
-    }
-    idx++;
-  }
-  return std::string::npos;
-}
-
-void StringRange::RestartCapture() {
-  start_ = data_;
-  end_ = data_;
-}
-
-StringRange StringRange::GetCaptured() {
-  return StringRange(start_, end_ - start_);
-}
-
-TypesWrapper& TypesWrapper::GetTypesWrapper() {
-  static TypesWrapper types;
-  return types;
-}
-
-std::unordered_set<std::string>& TypesWrapper::GetAllowedDataTypes() {
-  return allowed_data_types_;
-}
-
-std::unordered_map<std::string, int>& TypesWrapper::TypeStrToTensorDataType() {
-  return type_str_to_tensor_data_type_;
-}
-
-std::unordered_map<int, std::string>& TypesWrapper::TensorDataTypeToTypeStr() {
-  return tensor_data_type_to_type_str_;
-}
-
-TypesWrapper::TypesWrapper() {
-  // DataType strings. These should match the DataTypes defined in onnx.proto
-  type_str_to_tensor_data_type_["float"] = TensorProto_DataType_FLOAT;
-  type_str_to_tensor_data_type_["float16"] = TensorProto_DataType_FLOAT16;
-  type_str_to_tensor_data_type_["double"] = TensorProto_DataType_DOUBLE;
-  type_str_to_tensor_data_type_["int8"] = TensorProto_DataType_INT8;
-  type_str_to_tensor_data_type_["int16"] = TensorProto_DataType_INT16;
-  type_str_to_tensor_data_type_["int32"] = TensorProto_DataType_INT32;
-  type_str_to_tensor_data_type_["int64"] = TensorProto_DataType_INT64;
-  type_str_to_tensor_data_type_["uint8"] = TensorProto_DataType_UINT8;
-  type_str_to_tensor_data_type_["uint16"] = TensorProto_DataType_UINT16;
-  type_str_to_tensor_data_type_["uint32"] = TensorProto_DataType_UINT32;
-  type_str_to_tensor_data_type_["uint64"] = TensorProto_DataType_UINT64;
-  type_str_to_tensor_data_type_["complext64"] = TensorProto_DataType_COMPLEX64;
-  type_str_to_tensor_data_type_["complext128"] =
-      TensorProto_DataType_COMPLEX128;
-  type_str_to_tensor_data_type_["string"] = TensorProto_DataType_STRING;
-  type_str_to_tensor_data_type_["bool"] = TensorProto_DataType_BOOL;
-
-  for (auto& str_type_pair : type_str_to_tensor_data_type_) {
-    tensor_data_type_to_type_str_[str_type_pair.second] = str_type_pair.first;
-    allowed_data_types_.insert(str_type_pair.first);
-  }
-}
-} // namespace Utils
+    namespace Utils {
```

Comment:
can you run clang-format before landing to keep changes at bay?

Reply:
sure, and done.